hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotUtils.java

```diff
@@ -101,7 +102,7 @@ public static void checkSnapshotDirExist(File checkpoint)
     if (!checkpoint.exists()) {
       throw new OMException("Unable to load snapshot. " +
           "Snapshot checkpoint directory '" + checkpoint.getAbsolutePath() +
-          "' does not exists.", FILE_NOT_FOUND);
+          "' does not exists.", TIMEOUT);
```

Comment:
```suggestion
          "' does not exist yet. Please wait before retrying.", TIMEOUT);
```

Reply:
Please change the `TIMEOUT` in javadoc as well.

Reply:
done